build/x86_64_centos8/remi.repo

```diff
@@ -0,0 +1,9 @@
+[remi]
+name=Remi's RPM repository for Enterprise Linux $releasever - $basearch
```

Comment:
Please document the need for this repo. How do we know we can trust it?

Reply:
This person (Remi Collet) works for Red Hat, as far as I can tell, and is providing access to packages that aren't yet available in EPEL 8. It's recommended in a number of places online, and seems to be used by quite a few people — see [this answer on serverfault](https://serverfault.com/questions/946133/rhel-centos-is-remi-repo-safe-for-for-audits-etc) and https://rpms.remirepo.net/. The plan is to remove this repo once the packages appear in EPEL 8. Added a comment.

Reply:
Cool, thanks!